src/App.js

```diff
@@ -29,26 +21,66 @@ class App extends React.Component {
     return wordDisplay.toString();
   };
 
-  // Insert form callback functions handleChange and handleSubmit here
-
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <h1>Guess The Word 🚀</h1>
-          <h3>Word Display</h3>
-          {this.generateWordDisplay()}
-          <h3>Guessed Letters</h3>
-          {this.state.guessedLetters.length > 0
-            ? this.state.guessedLetters.toString()
-            : "-"}
-          <h3>Input</h3>
-          {/* Insert form element here */}
-          Todo: Insert form element here
-        </header>
-      </div>
-    );
-  }
-}
+  const handleChange = (event) => {
+    setFormData(event.target.value);
+  };
+
+  const isCorrectWord = () => {
+    for (const letter of currWord) {
+      if (!guessedLetters.includes(letter)) {
+        return false;
+      }
+    }
+    return true;
+  };
 
-export default App;
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (formData.length > 1) {
+      alert("Please only enter 1 letter");
+    } else {
+      setGuessedLetters([...guessedLetters, formData]);
+      setNumOfGuesses((prevGuess) => prevGuess - 1);
+      setFormData((prevFormData) => "");
```

Comment:
no need to declare the parameters, if not in use. 
```suggestion
      setFormData(() => "");
```